src/imagery/i.sar/i.sar.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+<head>
+<title>i.sar toolset - GRASS GIS manual</title>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="grassdocs.css" type="text/css">
+</head>
+<body bgcolor="white">
+<div id="container">
+
+<a href="https://grass.osgeo.org/grass-stable/manuals/index.html"><img src="grass_logo.png" alt="GRASS logo"></a>
+<hr class="header">
+
+<h2>NAME</h2>
+
+<em><b>i.sar</b></em> - Toolset to process SAR products.
+
+<h2>KEYWORDS</h2>
+<a href="https://grass.osgeo.org/grass-stable/manuals/imagery.html">imagery</a>,
+<a href="https://grass.osgeo.org/grass-stable/manuals/topic_import.html">import</a>,
+<a href="keywords.html#satellite">satellite</a>
+
+<!-- meta page description: Toolset to download and import Landsat products -->
+<h2>DESCRIPTION</h2>
+
+The <em>i.sar</em> toolset consists of three modules:
+
+<dl>
+  <dt><a href="i.sar.speckle/i.sar.speckle.html">i.sar.speckle</a></dt>
+    <dd>A simple procedure to reduce speckle noise in SAR images</dd>
+  <dt><a href="i.sar.amplitude/i.sar.amplitude.html">i.sar.amplitude</a></dt>
+    <dd>Module used to calculate an amplitude image from real and imaginary bands</dd>
+  <dt><a href="i.sar.pauliRGB/i.sar.pauliRGB.html">i.sar.pauliRGB</a></dt>
+    <dd>Module used to generate the three bands that form the Pauli RGB SAR composition</dd>
+</dl>
+
+
+<h2>AUTHORS</h2>
+Margherita Di Leo<br>
+Santiago Seppi, CONAE, Argentina
+
+<h2>SOURCE CODE</h2>
```

Comment:
I would remove this line, it may added automatically in the future, alternatively add manually the correct link to online documentation.